incubator/vault/templates/deployment.yaml

```diff
@@ -26,17 +36,34 @@ spec:
         {{- end }}
         ports:
         - containerPort: {{ .Values.service.port }}
+          name: api
+        - containerPort: 8201
```

Comment:
I didn't need to expose this port.
Is there any particular reason?

Sorry, i've not been using my vault cluster much, so I may missed this.

Reply:
It is part of the HA configuration docs. When you do `vault status` do you see the something like this that includes the `8201` in the `HA Cluster` section?

```
Key             Value
---             -----
Seal Type       shamir
Sealed          false
Total Shares    1
Threshold       1
Version         0.9.0
Cluster Name    vault-cluster-8c67b84a
Cluster ID      7f94daf3-92b8-56b9-2a61-15dccc743f6e
HA Enabled      true
HA Cluster      https://100.96.5.71:8201
HA Mode         active
```

Reply:
hmm, mine is a bit different:
```
Type: shamir
Sealed: false
Key Shares: 5
Key Threshold: 3
Unseal Progress: 0
Unseal Nonce: 
Version: 0.9.0
Cluster Name: vault-cluster-598d2444
Cluster ID: ec12aad8-bb73-c4fd-ddf1-102d3659fae7

High-Availability Enabled: true
	Mode: active
	Leader Cluster Address: https://10.60.0.10:8201
```

Reply:
Ah the minor formatting diff is just different versions of the vault CLI (looks like that output changed in 0.9.2). The point being that `8201` needs to be open.

Reply:
weird, cluster is functional, but it makes sense. I will debug and see. ( i think my pod-1 can reach the other pod-2 on 8201 even not been open)

Reply:
Just a side note, i will add port `8201` at my deployment.